llpc/builder/llpcBuilderCommon.h

```diff
@@ -0,0 +1,56 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Copyright (c) 2020 Advanced Micro Devices, Inc. All Rights Reserved.
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in all
+ *  copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ *
+ **********************************************************************************************************************/
+/**
+ ***********************************************************************************************************************
+ * @file  llpcBuilderCommon.h
+ * @brief LLPC header file: contains common interface types in the LGC interface
+ ***********************************************************************************************************************
+ */
+#pragma once
+
+#include <stdint.h>
+
+namespace Llpc
+{
+
+/// Enumerates LLPC shader types.
+enum ShaderType : uint32_t
+{
+    ShaderTypeVertex = 0,                                ///< Vertex shader
+    ShaderTypeTessControl,                               ///< Tessellation control shader
+    ShaderTypeTessEval,                                  ///< Tessellation evaluation shader
+    ShaderTypeGeometry,                                  ///< Geometry shader
+    ShaderTypeFragment,                                  ///< Fragment shader
+    ShaderTypeCompute,                                   ///< Compute shader
+    ShaderTypeCount,                                     ///< Count of shader stages
```

Comment:
In comments, stages -> types for this commits

Reply:
Ditto